### PR TITLE
Add product owner role description and links / details to applications

### LINKED ISSUE #19: Document product owner role
Write up what it means to be a PO for one of our applications. In the process I expect we'll want to talk about what we need / expect / wish for from POs.

### LINKED ISSUE #20: Create a list of applications with github links, tech leads, and POs
Can move the list out out of the "tech leads" page and use it as a starting point.

### PR DESCRIPTION
**Pull request by hackartisan**
Do not merge before #21 as this branch is based on that one.

closes #19
closes #20

### CODE REVIEW

File: applications.md [hunk 1]
```diff
@@ -1,13 +1,45 @@
...
-* PULMap: Eliot
-* PULBot/Heaven: Trey, transfer to Christina
-* LAE-Blacklight: Trey, transfer to Anna
-* Cicognara-Rails: Esme, transfer to Trey
-* PUL-Solr: Nikitas, transfer to Eliot
-* DSpace: James
-* Plantain: James
+* Bibdata, aka Marc Liberation
+  * Github repository: [https://github.com/pulibrary/marc_liberation]
+  * [Orangelight / Marc Liberation Zenhub board](https://app.zenhub.com/workspaces/orangelightmarcliberation-571691cab409d8d821b873be/board?repos=21954918)
```
Comment (jrgriffiniii) [inline applications.md:4]: 👍 for the links to the Zenhub boards

**Review by jrgriffiniii — APPROVED** [review]

File: product_owners.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Product Owners
+
+We strive for each of our applications to have a product owner (PO). The product
```
Comment (jrgriffiniii) [inline product_owners.md:3]: It's a pedantic point, but there are going to be cases where we are required to have multiple product owners for a given project.

Comment (hackartisan) [reply product_owners.md:3]: I wasn't sure I should get into all the details of why we prefer 1 and the exceptions, etc. If you have some wording you'd like to propose please go for it!

File: applications.md [hunk 3]
```diff
@@ -1,13 +1,45 @@
 ## Applications and their Technical Leads
```
Comment (tpendragon) [inline applications.md:1]: Can we put known product owners here too? Or at least leave a space for them to be filled in.

**Review by tpendragon — CHANGES_REQUESTED** [review]

File: product_owners.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+owner is usually a key stakeholder for the application who can collaborate
+closely with us to guide development. In working with our team, product owners
+should be familiar with our [work cycles](/work_cycles.md) and our [technical lead role](/technical_leads.md).
+
+The exact practices of each product owner will very according to the application and the
```
Comment (kelea99) [inline product_owners.md]: type-o. vary vs. very

File: product_owners.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Product Owners
+
+We strive for each of our applications to have a product owner (PO). The product
+owner is usually a key stakeholder for the application who can collaborate
+closely with us to guide development. In working with our team, product owners
```
Comment (kelea99) [inline product_owners.md]: wording here. perhaps saying the same thing a bit differently? up to you.  "We find that product owners who are familiar with our work cycles...etc....collaborate more effectively with our team." or something like that?

File: product_owners.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+should be familiar with our [work cycles](/work_cycles.md) and our [technical lead role](/technical_leads.md).
+
+The exact practices of each product owner will very according to the application and the
+individuals themselves. Our hope is that product owners will be able to:
```
Comment (kelea99) [inline product_owners.md]: I would nix the "our hope is that".  I would keep the accountability equal between Product owners and DRDS, as you have below. you don't hope to, you will work to--and we should give you and the application that same dedication.  you could say "Generally, product owners will work to/be able to:"

**Review by kelea99 — COMMENTED** [review]
Hello, everyone! I think it looks great an have made only a couple of suggestions to the PO part here.

File: applications.md (hunk 3 above)
Comment (kelea99) [reply applications.md:1]: I think that is a good idea

Comment (hackartisan) [reply applications.md:1]: I can put a few. Maybe tech leads can fill them in once this is merged, too.

File: product_owners.md (hunk 5 above)
Comment (hackartisan) [reply product_owners.md]: I'll work on this thank you!

File: product_owners.md (hunk 6 above)
Comment (hackartisan) [reply product_owners.md]: Will work on this also, thanks for pointing it out. I added the DRDS responsibilities later.

File: applications.md (hunk 3 above)
Comment (hackartisan) [reply applications.md:1]: (I am thinking, in some cases there may be de facto product owners but I am not sure they are familiar with that term / role. We may want to touch base with people as we go along)

File: applications.md [hunk 7]
```diff
@@ -1,13 +1,47 @@
...
+  * Github repository: [https://github.com/pulibrary/pul_solr]
+  * Tech lead: [Nikitas](https://github.com/tampakis), transfer to [Eliot](https://github.com/eliotjordan)
+* DSpace
+  * Github repository: [https://github.com/pulibrary/dspace]
+  * Tech lead: [James](https://github.com/jrgriffiniii)
+* Plantain (arclight)
+  * Github repository: [https://github.com/pulibrary/plantain]
+  * [Plantain Zenhub board](https://app.zenhub.com/workspaces/plantain-5da4b7d9f037f100019dba23/board?repos=157741631)
+  * Tech lead: [James](https://github.com/jrgriffiniii)
+  * Product owners: [Faith](https://github.com/faithc) and [Amanda](https://library.princeton.edu/staff/amandapf)
```
Comment (hackartisan) [inline applications.md]: I used a staff directory page here in lieu of a github account. does that seem okay?

File: product_owners.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
+# Product Owners
+
+We strive for each of our applications to have a product owner (PO). The product
+owner is usually a key stakeholder for the application who can collaborate
+closely with us to guide development. Documentation of our team's [work cycles](/work_cycles.md) and [technical lead role](/technical_leads.md) should give product owners useful context for this collaboration.
```
Comment (kelea99) [inline product_owners.md:5]: sounds great!

File: product_owners.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+The exact practices of each product owner will vary according to the application and the
+individuals themselves. Generally, product owners will work to:
+
+* Communicate with other stakeholders and users as-needed, either ad-hoc as a committee.
+* Participate during relevant work cycles by attending the planning meeting and
+  wrap-up meeting when the PO's application is a focus of the cycle.
+* Work with the technical lead to prioritize and specify issues. It is helpful
```
Comment (kelea99) [inline product_owners.md]: I would say its more than helpful for the PO to have a GitHub account. Perhaps changing the wording a bit? such as: [while not required], Creating/Having a GitHub account will enable/empower the PO to be a more integrated part of the overall process with our team. or something like that?

Comment (kelea99) [reply product_owners.md]: maybe even nix the "while not required" part...

Comment (hackartisan) [reply product_owners.md]: Okay I updated that; see what you think!

Comment (kelea99) [reply product_owners.md]: Looks good, @hackmastera !!

File: applications.md (hunk 7 above)
Comment (kelea99) [reply applications.md]: Seems like the best idea. That way you can ensure you can add a link for everyone.

Comment (escowles) [reply applications.md]: You can change it to @apferrar

Comment (hackartisan) [reply applications.md]: done, thanks!

**Review by tpendragon — APPROVED** [review]

**Pull request merged**